types/react-json/react-json-tests.tsx

```diff
@@ -0,0 +1,22 @@
+/// <reference types="node" />
+
+import * as React from 'react';
+const Json = require('react-json');
```

Comment:
Change this to an `import Json = require('react-json')`. There's no point in writing a `.d.ts` if you're just going to use an untyped `require`.